packages/react-native/src/primitives/Icon/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ColorValue, ImageProps, ImageStyle } from 'react-native';
+
+export interface IconStyles {
+  icon: ImageStyle;
+}
+
+export interface IconProps extends ImageProps {
+  /**
+   * Whether the icon is animated
+   */
+  animated?: boolean;
+
+  /**
+   * Color to be applied to the icon
+   */
+  color?: ColorValue;
+
+  /**
+   * Icon size
+   */
+  size?: number;
```

Comment:
What happens if a customer does not pass something to the `size` prop?

Also, do we want to support `'small' | 'medium' | 'large'` sizes?

Reply:
Yes. there is a Size type on the Radio currently, will follow up.